From c3889d60cee5a5d7fa215cdd442b1997d5239170 Mon Sep 17 00:00:00 2001
From: Andy Whitcroft <apw@canonical.com>
Date: Fri, 3 Dec 2010 09:51:33 +0000
Subject: [PATCH 0007/1767] UBUNTU: SAUCE: (no-up) add support for installed
 header files to ubuntu directory

BugLink: http://bugs.launchpad.net/bugs/684666

We need the aufs headers in the linux-libc-headers, add support for
including files from the ubuntu include directory.

Signed-off-by: Andy Whitcroft <apw@canonical.com>
---
 Makefile | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/Makefile b/Makefile
index 802be10..d7a4b1b 100644
--- a/Makefile
+++ b/Makefile
@@ -1060,6 +1060,7 @@ headers_install: __headers
 	  $(error Headers not exportable for the $(SRCARCH) architecture))
 	$(Q)$(MAKE) $(hdr-inst)=include/uapi
 	$(Q)$(MAKE) $(hdr-inst)=arch/$(hdr-arch)/include/uapi/asm $(hdr-dst)
+	$(Q)$(MAKE) $(hdr-inst)=ubuntu/include dst=include oldheaders=
 
 PHONY += headers_check_all
 headers_check_all: headers_install_all
@@ -1069,6 +1070,7 @@ PHONY += headers_check
 headers_check: headers_install
 	$(Q)$(MAKE) $(hdr-inst)=include/uapi HDRCHECK=1
 	$(Q)$(MAKE) $(hdr-inst)=arch/$(hdr-arch)/include/uapi/asm $(hdr-dst) HDRCHECK=1
+	$(Q)$(MAKE) $(hdr-inst)=ubuntu/include dst=include oldheaders= HDRCHECK=1
 
 # ---------------------------------------------------------------------------
 # Kernel selftest
-- 
1.9.3

